content/en/aws/opensearch/index.md

```diff
@@ -0,0 +1,415 @@
+...
+```
+
+and after some time, you should see that the `Processing` state of the domain is set to `false`:
+
+{{< command >}}
+$ awslocal opensearch describe-domain --domain-name my-domain | jq ".DomainStatus.Processing"
+false
+{{< / command >}}
+
+## Interact with the cluster
+
+You can now interact with the cluster at the cluster API endpoint for the domain, in this case `http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566`.
+
+For example:
+
+{{< command >}}
+$ curl http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566
+{
+  "name" : "host-pc",
+  "cluster_name" : "opensearch",
+  "cluster_uuid" : "DMN-2TlwRkuhMH4aRRqrkA",
+  "version" : {
+    "distribution" : "opensearch",
+    "number" : "1.1.0",
+    "build_type" : "tar",
+    "build_hash" : "15e9f137622d878b79103df8f82d78d782b686a1",
+    "build_date" : "2021-10-04T21:29:03.079792Z",
+    "build_snapshot" : false,
+    "lucene_version" : "8.9.0",
+    "minimum_wire_compatibility_version" : "6.8.0",
+    "minimum_index_compatibility_version" : "6.0.0-beta1"
+  },
+  "tagline" : "The OpenSearch Project: https://opensearch.org/"
+}
+{{< / command >}}
+
+Or the health endpoint:
+
+{{< command >}}
+$ curl -s http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/_cluster/health | jq .
+{
+  "cluster_name": "opensearch",
+  "status": "green",
+  "timed_out": false,
+  "number_of_nodes": 1,
+  "number_of_data_nodes": 1,
+  "discovered_master": true,
+  "active_primary_shards": 0,
+  "active_shards": 0,
+  "relocating_shards": 0,
+  "initializing_shards": 0,
+  "unassigned_shards": 0,
+  "delayed_unassigned_shards": 0,
+  "number_of_pending_tasks": 0,
+  "number_of_in_flight_fetch": 0,
+  "task_max_waiting_in_queue_millis": 0,
+  "active_shards_percent_as_number": 100
+}
+{{< / command >}}
+
+
+## Advanced topics
+
+### Endpoints
+
+There are two configurable strategies that govern how domain endpoints are created. The strategy can be configured via the `OPENSEARCH_ENDPOINT_STRATEGY` environment variable.
+
+| Value | Format | Description |
+| - | - | - |
+| `domain` | `<domain-name>.<region>.opensearch.localhost.localstack.cloud:4566` | This is the default strategy that uses the `localhost.localstack.cloud` domain to route to your localhost |
+| `path` | `localhost:4566/opensearch/<region>/<domain-name>` | An alternative that can be useful if you cannot resolve LocalStack's localhost domain |
+
+{{< alert >}}
+**Note**: Please be aware that - in contrast to the [Elasticsearch Service]({{< ref "elasticsearch" >}}) - the OpenSearch Service does _not_ support the endpoint strategy `off`.
```

Comment:
I think a double dash is needed here
```suggestion
**Note**: Please be aware that -- in contrast to the [Elasticsearch Service]({{< ref "elasticsearch" >}}) -- the OpenSearch Service does _not_ support the endpoint strategy `off`.
```

Reply:
Good point, we should use a dash instead of a hyphen here.